packages/gateway/src/features/systems/schema.graphql

```diff
@@ -59,4 +66,5 @@ type Mutation {
   reactivateSystem(clientId: ID!): System
   deactivateSystem(clientId: ID!): System
   registerSystem(system: SystemInput): SystemSecret
+  refreshSystemClientSecret(clientId: String!): RefreshSystemClientSecret
```

Comment:
```suggestion
  refreshSystemClientSecret(clientId: String!): SystemSecret
```
I think it would be better if we reusethe SystemSecret payload here. That way we won't need to define a new one